src/main/java/nl/hsac/fitnesse/fixture/util/HtmlCleaner.java

```diff
@@ -20,11 +21,16 @@
     public String getUrl(String htmlLink) {
         String result = htmlLink;
         if (htmlLink != null) {
-            Matcher matcher = PATTERN.matcher(htmlLink);
-            if (matcher.matches()) {
-                String href = matcher.group(1);
+            Matcher linkMatcher = LINKPATTERN.matcher(htmlLink);
+            Matcher imgMatcher = IMAGEPATTERN.matcher(htmlLink);
+            if (linkMatcher.matches()) {
+                String href = linkMatcher.group(1);
                 href = StringEscapeUtils.unescapeHtml4(href);
-                result = href + matcher.group(3);
+                result = href + linkMatcher.group(3);
+            }
```

Comment:
Really like the support for  <img>, that FitNesse makes of images in the files section.

Please place the '}' and 'else' on the same line.